azurerm/data_source_netapp_snapshot_test.go

```diff
@@ -0,0 +1,43 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func TestAccDataSourceAzureRMNetAppSnapshot_basic(t *testing.T) {
+	dataSourceName := "data.azurerm_netapp_snapshot.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
```

Comment:
This will need to be updated to match the existing tests:
```
data := acceptance.BuildTestData(t, "data.azurerm_netapp_snapshot", "test")
```

Reply:
fixed